src/main/java/net/sf/jabref/remote/DBMSType.java

```diff
@@ -0,0 +1,62 @@
+/*  Copyright (C) 2003-2016 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.remote;
+
+/**
+ * Enumerates all supported database systems (DBMS) by JabRef.
+ */
+public enum DBMSType {
+
+    MYSQL("MySQL"),
```

Comment:
What about extending the enum with the driver name as 2nd parameter?
e.g. `MYSQL(MySql, "com.com.mysql.jdbc")`
That would be easier to add new DBs and Drivers/change driver names.